chef/cookbooks/bcpc/recipes/nova-head.rb

```diff
@@ -405,3 +405,15 @@
   retries 30
   command 'openstack compute service list'
 end
+
+cron 'nova-manage db archive' do
+  action  :create
+  minute  node['bcpc']['nova']['db-archive']['cron_minute']
+  hour    node['bcpc']['nova']['db-archive']['cron_hour']
+  weekday node['bcpc']['nova']['db-archive']['cron_weekday']
+  day     node['bcpc']['nova']['db-archive']['cron_day']
+  month   node['bcpc']['nova']['db-archive']['cron_month']
+  user    'root'
+  command '/usr/local/bcpc/bin/if_leader nova-manage db archive_deleted_rows --until-complete --verbose 2>&1 | logger -t nova-db-archive-deleted-rows'
+  only_if { node['bcpc']['nova'].fetch('db-archive', {}).fetch('enabled', false) }
```

Comment:
```suggestion
  only_if { node['bcpc']['nova']['db-archive']['enabled'] }
```

now that db-archive is an attribute default, we can drop the check for it and look directly at enabled